src/style.css

```diff
@@ -0,0 +1,92 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+button {
+  cursor: pointer;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+  margin-top: 120px;
+  width: 60%;
+}
+
+.group {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.recent-score {
+  display: flex;
+  gap: 16px;
+  align-items: center;
+  font-size: 22px;
+}
+
+h3 {
+  font-size: 20px;
+}
+
+li {
+  list-style: none;
+}
+
+h1 {
+  align-self: flex-start;
+}
+
+.refresh {
+  background: 0;
+  border: 2px solid #000;
+  padding: 8px 16px;
+}
+
+.score-table {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  border: 3px solid #000;
+  padding: 10px 16px;
+  width: 400px;
+}
+
+.score-holder {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 32px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  width: 100%;
+}
+
+input {
+  padding: 10px;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly notice that you should add these lines of CSS code to make the button at the right of the form as it is in the wireframe.

```css
width: fit-content;
align-self: end;
```